conf/uppmax.config

```diff
@@ -60,16 +60,22 @@ process {
   $hisat2_sortOutput {
     module = ['bioinfo-tools', 'samtools/1.3']
     cpus = { 4 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    memory = { 24.GB * task.attempt }
     time = { 8.h * task.attempt }
   }
   $rseqc {
     module = ['bioinfo-tools', 'rseqc/2.6.1', 'samtools/1.3']
-    cpus = { 8 * task.attempt }
-    memory = { 32.GB * task.attempt }
-    time = { 7.h * task.attempt }
+    cpus = { 1 * task.attempt }
```

Comment:
I think just `1` by itself should be fine? It won't help to add more..

Reply:
(Yes I know that it will request more CPUs anyway because of the memory requirement, but it's probably safer and other people may copy this config file)